node-tests/nodetest-runner.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var glob = require('glob');
+var Mocha = require('mocha');
+var Promise = require('ember-cli/lib/ext/promise');
```

Comment:
they should work with RSVP, internally ember-cli `should` be casting promises up before depending on any of its extensions. Otherwise users code would also fail (which we dont want)